mantidproject/mantid
# Added dockable tabs to workbench data analysis

**Description of work.**

When the Data Analysis interface was ported to workbench the dockable tabs feature didn't work in Qt5. This updates the implimentation of this feature to restore the ability to dock and undock tabs. Bringing the interface on workbench into feature parity with that on plot. I developed this on windows so would be good to get a tester using a different OS to check the docking/undocking works ok there as this is the sort of thing which can vary between OS's.



**To test:**
Open the Data Analysis interface on workbench. Go to any of the fit tabs and check you can undock and re-dock the Function Browser and the plot windows in a sensible manner.



*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
